src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  font-size: 16px;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  .product-image {
+    padding: 16px 0;
+    margin: 0 auto;
+
+    .image {
+      width: 160px;
+      height: 134px;
+      object-fit: fill;
+    }
+  }
+
+  .product-name {
+    padding: 18px 0 4px;
+
+    &__link {
+      font-size: 12px;
+      line-height: 12px;
+      font-weight: 500;
+      color: #{$main-color};
+      text-decoration: none;
+    }
+  }
+
+  .product-code {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: #{$second-color};
+  }
+
+  .product-rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px 0;
+
+    &__reviews {
+      margin: 0;
+      font-size: 10px;
+      line-height: 14px;
+      color: #{$main-color};
+    }
+
+    @for $index from 1 through 5 {
+      .active-stars-#{$index} .star:nth-child(-n+#{$index}) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  .product-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 5px 0 16px;
```

Comment:
the sum of padding-top here (5px) and padding-bottom in `product-rate` (16px) is not enough (should be 24px):
![image](https://user-images.githubusercontent.com/86923089/230165611-e55686fa-9c75-488c-96a5-15d16705b786.png)

Anyway it's not the best practice to set both: top and bottom spacing (use only bottom, or only top)

Reply:
isn't fixed. And to make an indent between elements you need to use margin instead of padding. So you will have something like this (pixel values are random)
```
.product__image {
  margin: 0 auto 40px;
  ....
}

.product__name {
  margin-bottom: 20px;
  ...
}

.product__code {
  margin-bottom: 10px;
  ...
}

```